src/components/checkbox/index.test.js

```diff
@@ -24,7 +24,13 @@ describe('Checkbox', () => {
     checked,
   }
 
-  const wrapper = shallow(<Checkbox {...props} />, { context: { theme } })
+  const wrapper = shallow(<CheckboxInput {...props} />, { context: { theme } })
```

Comment:
If you pop these in `beforeEach` or separate `describe` blocks for each test, it will avoid confusion of multiple mounted components in the same context.